rpms/s3/s3rpm.spec

```diff
@@ -210,6 +224,7 @@ rm -rf %{buildroot}
 /opt/seagate/cortx/auth/AuthServer-1.0-0.jar
 /opt/seagate/cortx/auth/AuthPassEncryptCLI-1.0-0.jar
 /opt/seagate/cortx/auth/startauth.sh
+/opt/seagate/cortx/auth/scripts/swupdate/merge.sh
```

Comment:
Write a comment in %files% section about config behavior of yum

Reply:
done